core/src/main/scala/chisel3/VerificationStatement.scala

```diff
@@ -41,7 +41,16 @@ object assert {
   )(
     implicit sourceInfo: SourceInfo,
     compileOptions:      CompileOptions
-  ): Assert = macro _applyMacroWithMessage
+  ): Assert = macro _applyMacroWithStringMessage
+
+  def apply(
```

Comment:
This needs ScalaDoc

Reply:
Added - used the one present already as a template and made slight modifications! Hope that's fine.